spec/jobs/address_proofing_job_spec.rb

```diff
@@ -1,18 +1,108 @@
 require 'rails_helper'
+require 'proofing/address_mock_client'
 
 RSpec.describe AddressProofingJob, type: :job do
-  it 'stores results' do
-    document_capture_session = DocumentCaptureSession.new(result_id: SecureRandom.hex)
-    encrypted_arguments = Encryption::Encryptors::SessionEncryptor.new.encrypt(
-      { applicant_pii: { phone: Faker::PhoneNumber.cell_phone } }.to_json,
+  let(:document_capture_session) { DocumentCaptureSession.new(result_id: SecureRandom.hex) }
```

Comment:
Does it make sense to create a factory for doc capture proofing sessions or is it not worth it?

Reply:
I'm OK with it as-is, but can add a factory if you feel strongly

Reply:
No strong feelings here